src/App.tsx

```diff
@@ -1,14 +1,35 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
-import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { TodoModal } from './components/TodoModal';
+import { getTodos } from './utils/getTodos';
+import { Todo } from './types/Todo';
+
+import { getPreparedTodos } from './utils/getPrepearedTodos';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isTodosLoading, setIsTodosLoading] = useState(false);
+  const [currentTodo, setCurrentTodo] = useState<Todo | null>(null);
+  const [query, setQuery] = useState('');
+  const [filteredType, setFilteredType] = useState('all');
```

Comment:
```suggestion
  const [filterType, setFilterType] = useState(FilterType.all);
```
Avoid magic values, create enum and use it